ansible/roles/logstash/templates/logstash-simple-2.conf

```diff
@@ -0,0 +1,150 @@
+
+        grok {
+        #2020-06-15 17:15:15,275 [myid:] - INFO  [NIOServerCxn.Factory:0.0.0.0/0.0.0.0:2181:NIOServerCnxnFactory@222] - Accepted socket connection from /172.31.66.109:55402
+        match => { "message" => "%{TIMESTAMP_ISO8601:TIMESTAMP} \[%{GREEDYDATA:ID}:]\ - %{LOGLEVEL:level} \[%{GREEDYDATA:location}\]\ - %{GREEDYDATA:message}" }
+
+        #2020-06-16 14:34:20,639 [myid:] - INFO  [main:NIOServerCnxnFactory@89] - binding to port 0.0.0.0/0.0.0.0:2181
+        #2020-06-16 14:34:20,623 [myid:] - INFO  [main:Environment@100] - Server environment:java.compiler=<NA>
+        match => {"message" => "%{TIMESTAMP_ISO8601:TIMESTAMP} \[%{GREEDYDATA:ID}:]\ - %{LOGLEVEL:level} \[%{GREEDYDATA:method}:%{GREEDYDATA:className}\]\ \- %{GREEDYDATA:message}"}
+
+        #2020-06-16 18:41:28,734 [myid:] - INFO  [ProcessThread(sid:0 cport:2181)::PrepRequestProcessor@653] - Got user-level KeeperException when processing sessionid:0x1000007cddd000b type:create cxid:0xb7ef zxid:0x164a txntype:-1 reqpath:n/a Error Path:/accumulo/4d65522e-62cd-4540-9270-2272a3db2f85/masters/tick Error:KeeperErrorCode = NodeExists for /accumulo/4d65522e-62cd-4540-9270-2272a3db2f85/masters/tick
+        #2020-06-16 14:34:20,622 [myid:] - INFO  [main:Environment@100] - Server environment:java.class.path=/home/centos/install/zookeeper-3.4.14/bin/../zookeeper-server/target/classes:/home/centos/install/zookeeper-3.4.14/bin/../build/classes:/home/centos/install/zookeeper-3.4.14/bin/../zookeeper-server/target/lib/*.jar:/home/centos/install/zookeeper-3.4.14/bin/../build/lib/*.jar:/home/centos/install/zookeeper-3.4.14/bin/../lib/slf4j-log4j12-1.7.25.jar:/home/centos/install/zookeeper-3.4.14/bin/../lib/slf4j-api-1.7.25.jar:/home/centos/install/zookeeper-3.4.14/bin/../lib/netty-3.10.6.Final.jar:/home/centos/install/zookeeper-3.4.14/bin/../lib/log4j-1.2.17.jar:/home/centos/install/zookeeper-3.4.14/bin/../lib/jline-0.9.94.jar:/home/centos/install/zookeeper-3.4.14/bin/../lib/audience-annotations-0.5.0.jar:/home/centos/install/zookeeper-3.4.14/bin/../zookeeper-3.4.14.jar:/home/centos/install/zookeeper-3.4.14/bin/../zookeeper-server/src/main/resources/lib/*.jar:/home/centos/install/zookeeper-3.4.14/bin/../conf:
+        match => {"message" => "%{TIMESTAMP_ISO8601:TIMESTAMP} \[%{GREEDYDATA:ID}:]\ - %{LOGLEVEL:level} \[%{GREEDYDATA:method}:%{GREEDYDATA:className}\]\ \- %{GREEDYDATA:message}\:%{PATH}"}
+
+        #2020-06-17 13:37:17,041 [myid:] - INFO  [main:ZooKeeperServerMain@98] - Starting server
+        match => {"message" => "%{TIMESTAMP_ISO8601:TIMESTAMP} \[%{GREEDYDATA:ID}:]\ - %{LOGLEVEL:level}  \[%{GREEDYDATA:method}:%{GREEDYDATA:className}\] - %{GREEDYDATA:message}"}
+
+        match => {"message" => "%{TIMESTAMP_ISO8601:TIMESTAMP} \[%{GREEDYDATA:ID}:]\ - %{LOGLEVEL:level} \[%{GREEDYDATA:method}:%{GREEDYDATA:className}\]\ \- %{GREEDYDATA:message}\ /%{IPORHOST}"}
+
+
+
+        }
+
+    }
+        }
+output {
+    elasticsearch {
+        hosts => ["http://leader1:9200"]
```

Comment:
Please see my other comment about not hard-coding hostnames. Instead use a reference to the appropriate host from the Ansible host inventory.

Reply:
I hard-coded the hostname here `http://leader1:9200` because I use this section to set up the hostnames for Elasticsearch. But if you have any suggestions for change here, please don't hesitate to let me know.

Reply:
Please see my previous comment [here](https://github.com/apache/fluo-muchos/pull/338#discussion_r459845654) for the suggested change.

side note: I see you hit the Resolve button on many of my comments - normally it is expected that the originator of the comments hits the Resolve button in GitHub. Otherwise I am unable to reconcile if the original comment was addressed adequately. In the future, can you avoid hitting Resolve on a comment that a reviewer raised, and let them review, reconcile and eventually resolve it themselves?

Reply:
I will create a separate issue for the hard-coded host names and variables.

Reply:
Created issue #370 for this

Reply:
This is an easy fix and we should not merge this work without addressing this and related comments re: hard-coding. In the interest of helping close things, I have addressed these in a [commit on my own fork](https://github.com/arvindshmicrosoft/fluo-muchos/commit/d2d6b2d20c36052796de5b908ca7bb2d477f9e4b) - could you please take a look at that, merge as appropriate and then push changes to this PR? It would help close these issues sooner than later, and I personally would want to see those addressed before we approve and merge this PR.